src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -1114,20 +1113,37 @@ private void rejectIfForCrownDependency(String postcode) throws CTPException {
           }
 
           if (countryCode != null) {
-            actualRegion =
+            // Use the region as specified by AI
+            ccRegion =
                 uk.gov.ons.ctp.integration.contactcentresvc.representation.Region.valueOf(
                     countryCode);
           }
         }
       }
     }
 
-    if (actualRegion == null) {
-      log.with(requestedRegion).debug("Falling back to using Serco provided region");
-      actualRegion = requestedRegion;
+    // Decide if we are using the Serco or the CC calculated region
+    uk.gov.ons.ctp.integration.contactcentresvc.representation.Region regionForNewCase = null;
+    if (ccRegion == null) {
+      // CC failed to find the region
+      log.with(sercoRegion)
```

Comment:
see comment below about godaddy logging. Might be best to add variable name for consistency.